src/modules/tracker.js

```diff
@@ -856,6 +860,47 @@ class Tracker {
 
     return true;
   }
+
+  /**
+   * Expose helper function to add order id to storage
+   */
+  addOrderIdRecord(orderId) { /* eslint-disable-line class-methods-use-this */
+    let purchaseEventStorage = JSON.parse(store.session.get(purchaseEventStorageKey));
+    const orderIdHash = CRC32.str(orderId.toString());
+
+    if (purchaseEventStorage) {
+
+      // If the order already exists, do nothing
+      if (purchaseEventStorage[orderIdHash]) {
+        return;
+      }
+
+      purchaseEventStorage[orderIdHash] = true;
+    } else {
+
+      // Create a new object map for the order ids
+      purchaseEventStorage = {
+        [orderIdHash]: true,
+      };
+    }
+
+    // Push the order id map into session storage
+    store.session.set(purchaseEventStorageKey, JSON.stringify(purchaseEventStorage));
+  }
+
+  /**
+   * Expose helper function to check existence of order id in storage
+   */
+  hasOrderIdRecord(orderId) { /* eslint-disable-line class-methods-use-this */
+    const purchaseEventStorage = JSON.parse(store.session.get(purchaseEventStorageKey));
+    const orderIdHash = CRC32.str(orderId.toString());
+
+    if (purchaseEventStorage && purchaseEventStorage[orderIdHash]) {
+      return true;
+    }
+
+    return null;
```

Comment:
To me, `null` is used to explicitly communicates that there is no value to return, but I would expect `false` to be returned by a function named `hasOrderIdRecord` if the `orderIdRecord` does not exist. Either return value is falsey, but using `false` instead of `null` here gets our helper function to the happy place of always returning a boolean value.